### DISCUSSION TITLE
About SMART bracket notation

### DISCUSSION THREAD

**Discussion by TAEUKHAM** [rdkit/rdkit#4475] [Q&A] [answered]
As far as I know in SMART notation, atoms in a bracket notation could be a future atom candidate.
For example [C,N] mean this atom could be either aliphatic C or aliphatic N and [c,n] mean this atom could be either aromatic C or aromatic N

However, I have encountered rdkit keep spitting error when I make the atom candidates in aromatic form.
In the first picture, rdkit successfully converts smart code into molucules. 
But in the second picture, rdkit gives error message when I have changed [C,N] into [c,n].

In the third picture, I've made the smart code to be either benzene or pyridine as "c1cc[c,n]cc1" and also gives error message.

![github q1](https://user-images.githubusercontent.com/88373558/131943593-d253eb8f-4619-4185-9621-5e2921d36772.png)

![github q2](https://user-images.githubusercontent.com/88373558/131943601-1368d021-4c8d-42de-b4c3-64f35a629a1e.png)

![github q3](https://user-images.githubusercontent.com/88373558/131943607-e65c7834-7b4c-49e3-9e24-5a9ceff5d1d1.png)

**Comment by greglandrum** [comment, marked as answer]
Hi @TAEUKHAM, this was being caused by the bug reported in #4496.
It's fixed and will be in the 2021.03.6 patch release, which will be released at the end of the month.
Thanks for bringing it to our attention!

**Reply by TAEUKHAM** [reply]
@greglandrum 
Thanks for the reply Greg! Happy to find out that it was a bug ahah
